backend/test/rpc-client.test.js

```diff
@@ -0,0 +1,43 @@
+import assert from 'node:assert'
+import { describe, it } from 'node:test'
+import { chainHeadTestData } from './test_data/chainHead.js'
+import { rawActorEventTestData } from './test_data/rawActorEvent.js'
+import { parse } from '@ipld/dag-json'
+import { getActorEvents, getActorEventsFilter, getChainHead } from '../lib/rpc-service/service.js'
+import { ClaimEvent } from '../lib/rpc-service/data-types.js'
+import { Value } from '@sinclair/typebox/value'
+
+describe('RpcApiClient', () => {
+  const makeRpcRequest = async (method, params) => {
+    switch (method) {
+      case 'Filecoin.ChainHead':
+        return parse(JSON.stringify(chainHeadTestData))
+      case 'Filecoin.GetActorEventsRaw':
+        return parse(JSON.stringify(rawActorEventTestData)).filter(e => e.height >= params[0].fromHeight && e.height <= params[0].toHeight)
+      default:
+        console.error('Unknown method')
+    }
+  }
+
+  it('test the retrieval of the chainHead', async () => {
```

Comment:
Let's use the BDD-style test naming. The line should read like an English sentence.

Proposed test name: "RpcApiClient retrieves the chainHead"

`it()` stands for `RpcApiClient` here (the text you have in the `describe` block above).

```suggestion
  it('retrieves the chainHead', async () => {
```

The English sentence:

It _(meaning RpcApiClient)_ retrieves the chainHead.